gradle.properties

```
@@ -47,7 +47,7 @@ stdlibAuthVersion=2.9.0
 stdlibJwtVersion=2.9.0
 stdlibOAuth2Version=2.9.0
 
-stdlibHttpVersion=2.9.0
+stdlibHttpVersion=2.9.5-20231012-153400-1a7cfca
```

Need to update the version after http release
```suggestion
stdlibHttpVersion=2.9.5
```